ra/ra.go

```diff
@@ -1284,7 +1297,7 @@ func (ra *RegistrationAuthorityImpl) issueCertificateInner(
 	csr *x509.CertificateRequest,
 	profileName string,
 	acctID accountID,
-	oID orderID) (*x509.Certificate, error) {
+	oID orderID) (*x509.Certificate, string, []byte, error) {
```

Comment:
The new return values don't have self-explanatory types, so the comment should explain what they are.